Framework/Crystal/src/SelectCellWithForm.cpp

```diff
@@ -97,6 +101,56 @@ Kernel::Matrix<double> SelectCellWithForm::DetermineErrors(std::vector<double> &
     return newUB1;
 }
 
+void SelectCellWithForm::ApplyTransform(Kernel::Matrix<double> &newUB, IPeaksWorkspace_sptr &ws, double tolerance,
+                                        int *num_indexed, double *average_error) {
```

Comment:
My C++ is a bit rusty but would it be better practice to pass by reference here (and in a few other places) e.g. 
`int &num_indexed`
as a null pointer is not valid?

Reply:
That would be fine.